pom.xml

```diff
@@ -54,7 +54,6 @@
     <module>runtime</module>
     <module>deployment</module>
     <module>integration-test</module>
```

Comment:
We need to rename this module to integration-test**s**, I guess that's why the automation missed it.integration-tests

Reply:
the module has been renamed